### PR TITLE
i18n(fr): add migrate-to-astro/from-nuxtjs.mdx

### PR DESCRIPTION
**Pull request by ArmandPhilippot**
#### Description (required)

* Translate in French [Migrating from NuxtJS](https://docs.astro.build/en/guides/migrate-to-astro/from-nuxtjs/) (`migrate-to-astro/from-nuxtjs.mdx`) added in #2232

#### Related issues & labels (optional)

- Suggested label: i18n

Since this page does not exist yet, I think we should wait for #8765 to be merged before (I'll update the translation once it's done) so I converted this PR as draft for now.

### CODE REVIEW

**Review by thomasbnt — APPROVED** [review]
LGTM!

File: src/content/docs/fr/guides/migrate-to-astro/from-nuxtjs.mdx [hunk 1]
```diff
@@ -0,0 +1,674 @@
+---
+title: Migration depuis NuxtJS
+description: Astruces pour migrer un projet NuxtJS existant vers Astro
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/migrate-to-astro/from-nuxtjs.mdx:3]:
```suggestion
description: Astuces pour migrer un projet NuxtJS existant vers Astro
```

**Review by dreyfus92 — COMMENTED** [review]
Awesome work @ArmandPhilippot, I just spotted a tiny typo 😁

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by thomasbnt — APPROVED** [review]
LGTM!

**Pull request merged**